omnigibson/object_states/object_state_base.py

```diff
@@ -198,7 +198,7 @@ def update_cache(self, get_value_args):
     def cache_info(self, get_value_args):
         """
         Helper function to cache relevant information at the current timestep.
-        Stores it under @self._cache[<KEY>]["info"]
+        Stores it under @self._cache [<KEY>]["info"]
```

Comment:
what's the point of this space?

Reply:
the @ cross reference does not work with the space

Reply:
and it outputs an error